classes/mcPy/McPy.py

```diff
@@ -2,20 +2,14 @@
 import multiprocessing
 import os
 import sys
+import psutil
 
 from .Parser import Parser
 import classes.Server as Server
 
 
 def get_available_core():
-    try:
-        avail_cores = len(os.sched_getaffinity(0))
-    except AttributeError:
-        # Fix for windows, which doesnt support getaffinity
-        logging.warning(
-            "Falling back to multiprocessing cpu_count to calc cores. Most likely getaffinity is not supported on "
-            "your OS")
-        avail_cores = multiprocessing.cpu_count()
+    avail_cores = psutil.cpu_count()
```

Comment:
If we specifically want to get the number of usable cores rather than all CPUs on the system, `psutil` has a different way:
```python
>>> psutil.cpu_count()
6
>>> len(psutil.Process().cpu_affinity())
1
```
Works on Windows, which was the main problem with `os.sched_getaffinity()`

Reply:
`os.sched_getaffinity()` worked on Windows, but didn't work on MacOS

Reply:
Ah okay

Reply:
I'm going to test what you want to change on MacOS, since it might be different.